templates/configmap.yaml

```diff
@@ -25,6 +25,12 @@ data:
     remote_logging = True
 {{- end }}
 
+    [logging]
+    colored_console_log = False
+{{- if .Values.elasticsearch.enabled }}
```

Comment:
We should do this for statsd too:

https://github.com/apache/airflow/blob/0d51a12e26cdc5b291e0b9855852576ad8e610f6/UPDATING.md#metrics-configuration-has-been-moved-to-new-section

Reply:
Oh yes, done.